client/payment-methods-map.tsx

```diff
@@ -208,16 +221,37 @@ const PaymentMethodInformationObject: Record<
 	},
 	afterpay_clearpay: {
 		id: 'afterpay_clearpay',
-		label: __( 'Afterpay', 'woocommerce-payments' ),
-		brandTitles: {
-			afterpay_clearpay: __( 'Afterpay', 'woocommerce-payments' ),
-		},
-		description: __(
-			// translators: %s is the store currency.
-			'Allow customers to pay over time with Afterpay. Available to all customers paying in %s.',
-			'woocommerce-payments'
-		),
-		icon: AfterpayIcon,
+		label:
+			'GB' === accountCountry
+				? __( 'Clearpay', 'woocommerce-payments' )
+				: __( 'Afterpay', 'woocommerce-payments' ),
+		brandTitles:
+			'GB' === accountCountry
+				? {
+						afterpay_clearpay: __(
+							'Clearpay',
+							'woocommerce-payments'
+						),
+				  }
+				: {
+						afterpay_clearpay: __(
+							'Afterpay',
+							'woocommerce-payments'
+						),
+				  },
```

Comment:
A slight change, since the object structure is the same across all countries, it's just the individual string that needs a different name
```suggestion
		brandTitles: {
			afterpay_clearpay:
				'GB' === accountCountry
					? __( 'Clearpay', 'woocommerce-payments' )
					: __( 'Afterpay', 'woocommerce-payments' ),
		},
```